### PR TITLE
Minor dashboard updates

### PR DESCRIPTION
**Pull request by gtsiolis**
This will:
1. Change font weight in some labels.
2. Add missing redirect URL for the `/upgrade-subscription` link triggered from the IDE.
3. ...

### CODE REVIEW

File: components/dashboard/src/App.tsx [hunk 1]
```diff
@@ -94,6 +94,9 @@ function App() {
                 <Route path={["/subscription", "/usage"]} exact>
                     <Redirect to="/plans"/>
                 </Route>
+                <Route path={["/upgrade-subscription"]} exact>
```
Comment (svenefftinge) [inline components/dashboard/src/App.tsx:97]: Could you add this to the redirect above, so we have all known aliases per destination in one list?

Comment (gtsiolis) [reply components/dashboard/src/App.tsx]: Nice catch! Didn't notice the redirect routes above. 🙈 

Moved in cebf7c8.

**Comment by gtsiolis** [comment]
Leaving the pending changes modal as is for now.

@svenefftinge could you take a look, these should be trivial changes and good to merge.

**Review by svenefftinge — APPROVED** [review]

**Pull request merged**